### DISCUSSION TITLE
Does the command line tool work with Bunch Beta?

### DISCUSSION THREAD

**Discussion by moffat** [ttscoff/bunch-docs#237] [Q&A]
I'm running Bunch Beta 1.4.9. I installed the bunch cli using `gem install bunchcli --pre` (although using `gem install bunchcli` seems to install the same version). When I check the configuration with 

```
bunch --show-config
```

The bunches dir is set to my home directory. Trying to change it with ` bunch --pref configDir=MYDIR` didn't change the directory. 

Perhaps bunchcli isn't supposed to be working with the beta version?

**Comment by ttscoff** [comment]
Yeah, I'm afraid the CLI is currently hardcoded to use Bunch and not 
Bunch Beta. If the beta is the only version of Bunch installed, though, 
it _should_ still work. It uses x-bunch:// instead x-bunch-beta://, but 
the beta responds to both URL handlers. Do you also have the stable 
Bunch installed on the system?

-Brett

**Comment by ttscoff** [comment]
I just updated the bunchcli gem to 1.1.14 and included a check to see if 
Bunch Beta is running, altering the URLs it calls appropriately if it 
is. Hopefully that helps the situation a bit.

**Reply by moffat** [reply]
Thanks, that worked!
